client/build-theme.js

```diff
@@ -255,9 +253,11 @@ function combineLanguage(theme_name, target_dir) {
       paths.reverse();
       var input = [];
       for (var lang_path of paths) {
-        if (lang_path.startsWith(THEMES_ROOT)) {
+        if (fs.existsSync('./' + lang_path)) {
+          // using child path relative to build-theme
           input.push(require('./' + lang_path));
         } else {
+          // using other absolute/relative path
```

Comment:
This handles the theme path being passed to the function: if it doesn't have path indicators it is assumed to be a child path to the current folder and `./` is added, otherwise it is used as-is.